### PR TITLE
fixes 1470 hosts with no report should not be negative

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/models/dashboard.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def percentage
   end
 
   def reports_missing
-    report[:total_hosts] - report[:good_hosts_enabled] - report[:bad_hosts_enabled] - report[:out_of_sync_hosts_enabled] - report[:pending_hosts_enabled] - report[:disabled_hosts]
+    Host.search_for('not has last_report and status.enabled = true').count
```
Comment (ohadlevy) [inline app/models/dashboard.rb]: this does not take into account the user allowed hosts, I think you need Host.my_hosts.search ..

Comment (ohadlevy) [inline app/models/dashboard.rb]: also, there might be a search string in the first place, so this patch will not take that into account as well

Comment (isratrade) [inline app/models/dashboard.rb]: ohadlevy, this is now fixed in a new commit

**Comment by ohadlevy** [comment]
merged - thanks!

**Pull request closed without merging**